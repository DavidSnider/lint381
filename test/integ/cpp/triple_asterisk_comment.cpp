// good
// *** bad ***
/* good */

foo

/* *** bad *** */
