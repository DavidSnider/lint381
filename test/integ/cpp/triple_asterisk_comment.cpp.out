triple_asterisk_comment.cpp:2:1: error: Remove triple-asterisk comments
// *** bad ***
^^^^^^^^^^^^^^
triple_asterisk_comment.cpp:7:1: error: Remove triple-asterisk comments
/* *** bad *** */
^^^^^^^^^^^^^^^^^
